src/main/java/hudson/plugins/jacoco/model/CoverageObject.java

```diff
@@ -203,7 +204,7 @@ public Coverage getLineCoverage() {
 	/**
 	 * Gets the build object that owns the whole coverage report tree.
 	 */
-	public abstract AbstractBuild<?,?> getBuild();
+	public abstract Run<?,?> getBuild();
```

Comment:
I would run a mile from the BCT.  What I am suggesting is you reach out to the plugin maintainer and ask them about breaking backwards API compatibility (and then if they agree do it properly :-) )